dex/testing/loadbot/go.mod

```diff
@@ -1,6 +1,8 @@
 module decred.org/dcrdex/dex/testing/loadbot
 
-go 1.19
+go 1.21.10
+
+toolchain go1.22.4
```

Comment:
There is no reason to require a minimum toolchain that is higher than the required version of go here as doing so would force all consumers to do so too.

This line should be removed.

The only exception would be is if one of the deps is doing it thus forcing yours to hoist it forward (which is exactly why you shouldn't do it to others either unless you _really_ need something from the newer version, but don't absolutely require it.

Reply:
go mod tidy is adding this for some reason starting in my commit before this one.

Reply:
Then one of the deps probably incorrectly has it and is forcing all consumers to hoist forward.  Sigh.

Reply:
Ok, it is sorted now. Whenever @martonp rebases on #2885 it will be fixed.